templates/triage_owner_rotations.html

```diff
@@ -19,6 +20,9 @@
                 <td>{{ row['component'] }}</td>
                 <td>{{ row['old_triage_owner'] }}</td>
                 <td>{{ row['new_triage_owner'] }}</td>
+                <td>
+                    <a href="{{ row['link_to_triage'] }}">View Triage</a>
```

Comment:
What do you think about making the component name clickable and pointing to Bugdash?

Reply:
Good idea, I've pushed these changes